src/Controller/AbstractActionController.php

```diff
@@ -85,26 +87,4 @@ public function onDispatch(MvcEvent $e)
 
         return $actionResponse;
     }
-
-    /**
-     * @deprecated please use the {@see \Zend\Mvc\Controller\Plugin\CreateHttpNotFoundModel} plugin instead: this
-     *             method will be removed in release 2.5 or later.
-     *
-     * {@inheritDoc}
-     */
-    protected function createHttpNotFoundModel(HttpResponse $response)
```

Comment:
While these methods _should_ have been removed for the 3.0 release, they were not, and, as such, doing so now is a backwards compatibility break. Please create a separate PR for these removals which we can tag for the 4.0 release (which will target PHP 7.1).

Thanks!